addons/mail/models/mail_followers.py

```diff
@@ -508,3 +509,20 @@ def _add_followers(self, res_model, res_ids, partner_ids, subtypes,
                         update[fol_id] = {'subtype_ids': update_cmd}
 
         return new, update
+
+    def _follower_fetch(self, domain, after=None, limit=100):
```

Comment:
This is globally a useless method, code can be embed in the two calls: if is just an if and a search. When possible avoid adding layer, it makes code hard to understand.